src/main/java/org/opentripplanner/graph_builder/module/DirectTransferGenerator.java

```diff
@@ -115,7 +116,13 @@ public void buildGraph() {
         LOG.debug("Linking stop '{}' {}", stop, ts0);
 
         for (RouteRequest transferProfile : transferRequests) {
-          RouteRequest streetRequest = transferProfile.copyAndPrepareForTransferRouting();
+          var streetRequest = transferProfile.clone();
+
+          // Make sure we use a generic request, without any specific fields set
+          streetRequest.setArriveBy(false);
+          streetRequest.setDateTime(Instant.ofEpochSecond(0));
+          streetRequest.setFrom(null);
+          streetRequest.setTo(null);
```

Comment:
I feel that this should this be encapsulated and named explcitly.

Reply:
Done, we can eventually get rid of this by adding arriveBy as a separate field to the AStarRequest. Bu I will do that another time.